terraform/main.tf

```diff
@@ -0,0 +1,24 @@
+resource "digitalocean_droplet" "website" {
+  image              = var.do_image
+  region             = var.do_region
+  name               = "website"
+  size               = var.do_size
+  private_networking = true
+  ssh_keys = [
+    "${var.ssh_fingerprint}"
```

Comment:
With latest terraform, you should be able to just use `var.ssh_fingerprint`(no need: `${}`).